src/fragments/guides/location-service/setting-up-your-app-js.mdx

```diff
@@ -47,9 +47,13 @@ In the following procedure, you’ll add authentication to your React app using
 
 **To add authentication to your React app**
 
-1. From your projects directory, run the following command, and answer the prompted questions as indicated.
+1. To do so, we'll initialize an Amplify project and add authentication. From your projects directory, run the following command, and answer the prompted questions as indicated.
```

Comment:
projects should be possessive here - 'project's'

Reply:
I'd cut "To do so, we'll " and just go with "Initialize an Amplify project and add authentication"